dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jaxws.common-2.3.feature

```diff
@@ -1,46 +1,66 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
In order to provide a truly comment set of bundles between the server and client containers, I've refactored the jaxws-2.3 feature files as well as the jaxwsClient-2.3 feature.

Reply:
And alphabetized the API entries